about.html

```diff
@@ -7,58 +7,65 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <header>
+    <nav>
+        <img src="img/lambda-black.png" alt="lambda logo">
+        <a href="index.html">Home</a>
```

Comment:
Great job linking back to the home page. Navigation is crucial for the users, you dont want them to get stuck on a page with no way of returning